build.gradle

```diff
@@ -273,6 +273,13 @@ task codeCoverageReport(type: JacocoReport) {
           //add source sets from all relevant submodules
           sourceSets subproject.sourceSets.main
         }
+        classDirectories.setFrom(files(classDirectories.files.collect {
+          fileTree(dir: it, exclude: [
```

Comment:
nit - can you add a comment explaining this is done to explode some classes from coverage by package?

Reply:
added